### DISCUSSION TITLE
Please help configure Flow Launcher

### DISCUSSION THREAD

**Discussion by kayanco** [Flow-Launcher/Flow.Launcher#2370] [Q&A]
Hi,

I need to use Flow Launcher in a specific manner:

![cPDdiVj](https://github.com/Flow-Launcher/Flow.Launcher/assets/33558698/5bf367c6-079b-4264-8606-da14a1d2ee61)

I want to **only** receive results from specific folders, with different criteria for each location, e.g.:

Location 1: certain extensions, depth 2
Location 2: just folders, depth 3

Can someone please help me?

**Comment by jjw24** [comment]
I think this maybe configurable via Everything instead
